bundle/src/main/java/com/adobe/acs/commons/httpcache/store/jcr/impl/writer/EntryNodeWriter.java

```diff
@@ -0,0 +1,120 @@
+package com.adobe.acs.commons.httpcache.store.jcr.impl.writer;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.Calendar;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import javax.jcr.Binary;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+import org.apache.jackrabbit.commons.JcrUtils;
+
+import com.adobe.acs.commons.httpcache.engine.CacheContent;
+import com.adobe.acs.commons.httpcache.keys.CacheKey;
+import com.day.cq.commons.jcr.JcrConstants;
+
+public class EntryNodeWriter
+{
+
+    private final Session session;
+    private final Node entryNode;
+    private final CacheKey cacheKey;
+    private final CacheContent cacheContent;
+    private final int expireTimeInSeconds;
+
+    public EntryNodeWriter(Session session, Node entryNode, CacheKey cacheKey, CacheContent cacheContent,
+            int expireTimeInSeconds){
+        this.session = session;
+        this.entryNode = entryNode;
+        this.cacheKey = cacheKey;
+        this.cacheContent = cacheContent;
+        this.expireTimeInSeconds = expireTimeInSeconds;
+    }
+
+    /**
+     * Populate the entry node with values
+     * @throws RepositoryException
+     */
+    public void write() throws RepositoryException, IOException
+    {
+        entryNode.setProperty("isCacheEntryNode", true);
```

Comment:
Should pull these constant property names out (saw them around in other code above) .. example.. `PN_IS_CACHE_ENTRY_NODE = "isCacheEntryNode"`

`PN` being the idiomatic prefix for **P**roperty **N**ames

Reply:
Alright, il rename the other property constants as well to PN_*.